packages/themeables/border/src/types.ts

```diff
@@ -0,0 +1,23 @@
+import { TColor } from 'colorido'
+
+export type TThemeableBorder = {
+  color: TColor,
+  topLeftRadius?: number,
```

Comment:
I may be missing something, but shouldn't the `borderRadius` type be a `string` for allowing consumers to switch between `%`, `px` and fraction values? 🤔

Reply:
It relates to the goal of the primitives and themeables. These are meant to serve as cross-platform abstractions, and with that in mind, the only absolute unit of measure that can be guaranteed to work is `px`. Fractional values are hopefully supported automatically since JS numbers are floats, although we can't be sure how rendering of those would be interpreted.

That said, a case can be made about `%`. I do not recall why we do not support `%` here.

Reply:
as for me the only usecase of `%`-radius is `50%`, and implementing that would imply that we don't know the dimensions, which is not the case. `%` itself is a "magic" side of CSS we've been trying to avoid, like `rem`/`em`/etc. if we allow `%` once then it  will leak into `width` values as well because "why not" and we'll end up in hell. I'd say that the end goal is to stick with pixels as much as we can as the only "real" values and skip all the CSS magic we don't control.

Reply:
@xaviervia the fraction values in [border-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) carry the units, so they would be `string` in JS, not floats, as far as I can understand.

---

@deepsweet 
Thanks for elaborating, that's an interesting take on relative units and CSS.
Though I feel I'm missing some more background on the library usage to fully understand what you mean here, so I'll hold on to it 🤐

Reply:
@atilafassina I misunderstood about the fraction values, I thought you meant values with decimal points. Yeah, then for the same reason as @deepsweet mentions we wouldn't be able to support those in here :(